backend/core/ai/test_category_query.py

```
@@ -0,0 +1,73 @@
+
+# SQL 캐싱 추가
+if result.get('generated_sql') and not result.get('error'):
+    saved_metric.sql_query = result['generated_sql']
+    saved_metric.sql_generated_at = datetime.now(UTC)
+    repository.update_metric(saved_metric.id, saved_metric)
+    print(f"\n✅ SQL 캐싱 완료: {result['generated_sql']}")
+
+# 저장된 히스토리 확인
+if result.get('history_id'):
+    history = repository.get_history(result['history_id'])
+    print(f"\n저장된 데이터:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)